### ISSUE TITLE
Issue with Etherscan

### ISSUE THREAD

**Issue by wighawag** [justinjmoses/eth-reveal#1]
```
ethers/providers/etherscan-provider.js:209
                    throw new Error('EtherscanProvider does not support blockTag for call');
```

This happen when calling revealer on a tx that failed, for example : 

`npx eth-reveal -h 0x5f44b08d30a3189a86a73f016890c52e462e9387fdfc5b571c3093312c0f3667`

**Comment by jjgonecrypto** [comment]
Thanks for reporting. I believe the issue is that Infura no longer supports `blockTag` and ethers by default uses Infura first then Etherscan - and it's the Etherscan error you see. 

I believe it's from this line: https://github.com/justinjmoses/eth-reveal/blob/master/lib/lookup.js#L160 which calls using a previous block to get the revert reason. 

This is an issue reported on ethers.js about it (even though it's not the library's fault): https://github.com/ethers-io/ethers.js/issues/715

One workaround would be to use an Infura archive node as provider, but that costs $50+250 USD p/m. 

Nevertheless, I should handle the error gracefully, and I'll fix that now.

**Comment by jjgonecrypto** [comment]
I tried wrapping the call in a `try/catch` but ran into this bug I reported on ethers-js: https://github.com/ethers-io/ethers.js/issues/729. I'll update here when I hear something.

**Comment by jjgonecrypto** [comment]
Actually, `blockNumber` isn't needed for this `call` at all. I published a fix `0.5.11`.
